monai/apps/auto3dseg/algorithms/templates/dints/configs/hyper_parameters.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# hyper-parameters
+amp: true
+bundle_root: null
+ckpt_path: "$@bundle_root + '/model_fold' + str(@fold)"
```

Comment:
@wyli 's API design in PR #4891 already splits the fold folders in the bundle_root. `'/model_fold' + str(@fold)` may be replaced by `'model'`